examples/jsm/nodes/core/NodeBuilder.js

```diff
@@ -210,25 +210,25 @@ class NodeBuilder {
 
 	}
 
-	getTexture( /* textureProperty, uvSnippet */ ) {
+	getTexture( /* texture, textureProperty, uvSnippet */ ) {
```

Comment:
Maybe we can try merging `getTexture` and `getCubeTexture` functions together now that we have `texture` passed as a parameter? (Similar as for VideoTexture you didn't created a separate `getVideoTexture` function but modified `getTexture`)